benchmarks/tool/compile_benchmarks.dart

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env dart
+
+import 'dart:io' show exit, Platform, Process, Directory;
+
+import 'package:args/args.dart' show ArgParser;
+import 'package:path/path.dart' as path;
+import 'package:pool/pool.dart' show Pool;
+
+Future<void> main(List<String> args) async {
+  final argParser = ArgParser()
+    ..addOption('target', mandatory: false, defaultsTo: 'jit,aot,js')
+    ..addOption('jobs', abbr: 'j', mandatory: false);
+
+  final parsedArgs = argParser.parse(args);
+
+  var jobs = Platform.numberOfProcessors;
+  if (parsedArgs['jobs'] != null) {
+    jobs = int.parse(parsedArgs['jobs']!);
+  }
+
+  var targets = <Target>{};
+  for (final targetStr in parsedArgs['target'].split(',')) {
+    switch (targetStr) {
+      case 'jit':
+        targets.add(jitTarget);
+        break;
+
+      case 'aot':
+        targets.add(aotTarget);
+        break;
+
+      case 'js':
+        targets.add(jsTarget);
+        break;
+
+      default:
+        print(
+            'Unsupported target: $targetStr. Supported targets: jit, aot, js');
+        exit(1);
+    }
+  }
+
+  // Arg list is immutable, clone it
+  var sourceFiles = List.from(parsedArgs.rest);
```

Comment:
nit: Prefer to use `final` for all variables that aren't reassigned (I believe there's analyzer lints you could enable for that)

Reply:
Done.